cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/03-grants-and-scholarships.html

```diff
@@ -0,0 +1,168 @@
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'GI Bill',
+                        'helper_text': 'We recommend checking out the <a href="https://www.va.gov/gi-bill-comparison-tool/" target="_blank" class="external-link">GI Bill&reg; Comparison Tool</a> too.' | safe,
+                        'id': 'grants__gibill',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="grant_gibill"'
+                    })
+                }}
+                {{
+                    input.render({
+                        'label': 'Other (Yellow Ribbon, etc)',
+                        'id': 'grants_serviceOther',
+                        'value': '0',
+                        'data_attribute': 'data-financial-item="grantserviceOther"'
+                    })
+                }}
+            </div>
+            {{
+                text_item.render({
+                    'label': 'Total grants',
+                    'value': '$99,999',
+                    'type': 'subtotal',
+                    'data_attribute': 'data-financial-item="total_grants"'
+                })
+            }}
+        </div>
+    </div>
+    <div class="grants-and-scholarships_content-section financial-items">
+        <div class="block block__mid">
+            <h3 class="header__border-bottom">Scholarships</h3>
+            {{
+                input.render({
+                    'label': 'State scholarship(s)',
+                    'id': 'scholarships__state',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="scholarship_state"'
+                })
+            }}
+            {{
+                input.render({
+                    'label': 'School scholarship(s)',
+                    'id': 'scholarships__school',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="scholarship_school"'
+                })
+            }}
+            {{
+                input.render({
+                    'label': 'Other scholarship(s)',
+                    'id': 'scholarships__other',
+                    'value': '0',
+                    'data_attribute': 'data-financial-item="scholarship_other"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Total scholarships',
+                    'value': '$99,999',
+                    'type': 'subtotal',
+                    'data_attribute': 'data-financial-item="total_scholarships"'
+                })
+            }}
+            {{
+                text_item.render({
+                    'label': 'Total grants and scholarships',
+                    'value': '$99,999',
+                    'type': 'total',
+                    'data_attribute': 'data-financial-item="total_grantsScholarships"'
```

Comment:
I think you could drop the quotes for the keys in these objects since none have spaces, if you wanted to. Run `gulp lint --fix` if you haven't already and it might have feedback here, maybe.